src/tools/web/clusterd/src/config.js

```diff
@@ -0,0 +1,2 @@
+export const PORT = process.env.PORT || 33334
```

Comment:
@markus2330 it is a sort-of standard to allow applications to be executed on a different port by specifying the `PORT` envvar (e.g. a lot of container systems work with this). does it really make sense to load it from elektra? maybe try to load it from the env var first, then elektra, then fall back to the default port?